cetmix_tower_server/models/cx_tower_plan_line.py

```
@@ -88,6 +92,23 @@ def _check_command_id(self):
+        """
+        Compute the preview of the command based on its action.
+        """
+        for line in self:
+            if line.action == "file_using_template":
+                line.command_code = (
+                    line.command_id.code or "No template code available"
+                )
+            elif line.action == "plan":
+                flight_plan_lines = line.command_id.flight_plan_id.line_ids
+                preview_lines = [line.name for line in flight_plan_lines]
+                line.command_code = "\n".join(preview_lines) if preview_lines else "No related lines available"
```

Ruff (0.8.2)

Line too long (111 > 88)

(E501)